python/hail/dataset.py

```diff
@@ -1811,28 +1788,31 @@ def filter_variants_list(self, input, keep=True):
 
         :param str input: Path to variant list file.
 
-        :return: A Variant Dataset including or excluding variants as specified
-
+        :return: a filtered dataset including or excluding variants as specified
+        :return: filtered dataset
         :rtype: :py:class:`.VariantDataset`
-
         """
 
         pargs = ['filtervariants', 'list',
                  '--keep' if keep else '--remove',
                  '-i', input]
-        return self.hc.run_command(self, pargs)
+        return self.hc._run_command(self, pargs)
+
+    def globals(self):
+        """Return global annotations as a python object.
+
+        :return: vds global annotations
```

Comment:
I'd say "Global annotations." or "Dataset global annotations."   VDS sounds like a file (and should be capitalized).